docs/windows.md

```diff
@@ -80,6 +81,8 @@ It is necessary to always:
 - Carry the OVMF firmware in the `--kernel` option
 - Add `kvm_hyperv=on` to the `--cpus` option
 
+In cases where the host processor supports address space &gt; 39 bits, it might be necessary to limit the address space. It can be done by appending the option `max_phys_bits=X` to the `--cpus` parameter, where `X` is the number of bits to be supported. Windows was tested to support at least 39-bit address space.
```

Comment:
Why does it need to be `&gt;` in markdown?

Reply:
It's just me being used to html, fixed.